### PR TITLE
Upgrade framework and packages  

### PR DESCRIPTION
**Pull request by MeladKamari**
Upgrade to .net8 
Upgrade some packages
add Directory.Build.props, Directory.Packages.props for manage globally packages and framework version

### CODE REVIEW

**Review by IEvangelist — APPROVED** [review]
This looks great, would you be willing to upgrade all the other `Orleans` samples?

**Comment by MeladKamari** [comment]
> @MeladKamari - This looks great, would you be willing to upgrade all the other `Orleans` samples?

@IEvangelist  - Yes i do it

**Comment by MeladKamari** [comment]
> @MeladKamari - This looks great, would you be willing to upgrade all the other `Orleans` samples?

@IEvangelist - Upgrade all projects in one PR?

File: orleans/Adventure/Directory.Packages.props [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Hosting" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Logging.Console" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.Orleans.Client" Version="7.2.3" />
+    <PackageVersion Include="Microsoft.Orleans.Sdk" Version="7.2.3" />
+    <PackageVersion Include="Microsoft.Orleans.Server" Version="7.2.3" />
```
Comment (IEvangelist) [inline orleans/Adventure/Directory.Packages.props:12]:
```suggestion
    <PackageVersion Include="Microsoft.Orleans.Client" Version="8.0.0" />
    <PackageVersion Include="Microsoft.Orleans.Sdk" Version="8.0.0" />
    <PackageVersion Include="Microsoft.Orleans.Server" Version="8.0.0" />
```

**Review by IEvangelist — APPROVED** [review]

**Comment by IEvangelist** [comment]
> > @MeladKamari - This looks great, would you be willing to upgrade all the other `Orleans` samples?
> 
> @IEvangelist - Upgrade all projects in one PR?

@MeladKamari - I was more or less asking if you'd be interested in doing a PR for the other Orleans samples as well?

**Comment by MeladKamari** [comment]
> > > @MeladKamari - This looks great, would you be willing to upgrade all the other `Orleans` samples?
> > 
> > 
> > @IEvangelist - Upgrade all projects in one PR?
> 
> @MeladKamari - I was more or less asking if you'd be interested in doing a PR for the other Orleans samples as well?

@IEvangelist - Yes , I will upgrade all samples

**Pull request merged**